cmd/influx/bucket.go

```diff
@@ -65,9 +64,9 @@ func newBucketService(f Flags) (platform.BucketService, error) {
 }
 
 func bucketCreateF(cmd *cobra.Command, args []string) error {
-	if bucketCreateFlags.orgID == "" && bucketCreateFlags.org == "" {
+	if bucketCreateFlags.organization.id == "" && bucketCreateFlags.organization.name == "" {
 		return fmt.Errorf("must specify org-id, or org name")
-	} else if bucketCreateFlags.orgID != "" && bucketCreateFlags.org != "" {
+	} else if bucketCreateFlags.organization.id != "" && bucketCreateFlags.organization.name != "" {
```

Comment:
you can make this a method on the organization type itself and call it here 👍

Reply:
ah - I did https://github.com/influxdata/influxdb/pull/16225/files#diff-0f706fc08b27c10d38fa0be84c6ebf3eR270 just forgot to call it. good catch!